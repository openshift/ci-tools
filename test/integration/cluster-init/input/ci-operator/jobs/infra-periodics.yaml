periodics:
- name: periodic-rotate-serviceaccount-secrets
  spec:
    containers:
    - env:
      - name: KUBECONFIG
        value: :/etc/build-farm-credentials/sa.config-updater.build01.config
      name: ""
      resources: {}
    volumes:
    - name: build-farm-credentials
      secret:
        items:
        - key: sa.config-updater.build01.config
          path: sa.config-updater.build01.config
        secretName: secret
- name: periodic-ci-secret-generator
  spec:
    containers:
    - env:
      - name: KUBECONFIG
        value: :/etc/build-farm-credentials/sa.config-updater.build01.config
      name: ci-secret-bootstrap
      resources: {}
    volumes:
    - name: build-farm-credentials
      secret:
        items:
        - key: sa.config-updater.build01.config
          path: sa.config-updater.build01.config
        secretName: secret
- name: periodic-ci-secret-bootstrap
  spec:
    containers:
    - env:
      - name: KUBECONFIG
        value: :/etc/build-farm-credentials/sa.config-updater.build01.config
      name: ci-secret-bootstrap
      resources: {}
    volumes:
    - name: build-farm-credentials
      secret:
        items:
        - key: sa.config-updater.build01.config
          path: sa.config-updater.build01.config
        secretName: secret
