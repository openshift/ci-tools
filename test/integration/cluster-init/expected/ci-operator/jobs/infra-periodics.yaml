periodics:
- name: periodic-rotate-serviceaccount-secrets
  spec:
    containers:
    - env:
      - name: KUBECONFIG
        value: :/etc/build-farm-credentials/sa.config-updater.build01.config:/etc/build-farm-credentials/sa.config-updater.newCluster.config
      name: ""
      resources: {}
    volumes:
    - name: build-farm-credentials
      secret:
        items:
        - key: sa.config-updater.build01.config
          path: sa.config-updater.build01.config
        - key: sa.config-updater.newCluster.config
          path: sa.config-updater.newCluster.config
        secretName: secret
- name: periodic-ci-secret-generator
  spec:
    containers:
    - env:
      - name: KUBECONFIG
        value: :/etc/build-farm-credentials/sa.config-updater.build01.config:/etc/build-farm-credentials/sa.config-updater.newCluster.config
      name: ci-secret-bootstrap
      resources: {}
    volumes:
    - name: build-farm-credentials
      secret:
        items:
        - key: sa.config-updater.build01.config
          path: sa.config-updater.build01.config
        - key: sa.config-updater.newCluster.config
          path: sa.config-updater.newCluster.config
        secretName: secret
- name: periodic-ci-secret-bootstrap
  spec:
    containers:
    - env:
      - name: KUBECONFIG
        value: :/etc/build-farm-credentials/sa.config-updater.build01.config:/etc/build-farm-credentials/sa.config-updater.newCluster.config
      name: ci-secret-bootstrap
      resources: {}
    volumes:
    - name: build-farm-credentials
      secret:
        items:
        - key: sa.config-updater.build01.config
          path: sa.config-updater.build01.config
        - key: sa.config-updater.newCluster.config
          path: sa.config-updater.newCluster.config
        secretName: secret
- agent: kubernetes
  cluster: app.ci
  decorate: true
  extra_refs:
  - base_ref: master
    org: openshift
    repo: release
  interval: 12h
  labels:
    ci.openshift.io/role: infra
  name: periodic-openshift-release-master-build01-apply
  spec:
    containers:
    - args:
      - --config-dir=clusters/build-clusters/build01
      - --as=
      - --KUBECONFIG=/etc/build-farm-credentials/KUBECONFIG
      - --confirm=true
      command:
      - applyconfig
      image: applyconfig:latest
      imagePullPolicy: Always
      name: ""
      resources:
        requests:
          cpu: 10m
      volumeMounts:
      - mountPath: /etc/build-farm-credentials
        name: build-farm-credentials
        readOnly: true
    serviceAccountName: config-updater
    volumes:
    - name: build-farm-credentials
      secret:
        items:
        - key: sa.config-updater.build01.config
          path: kubeconfig
        secretName: build-farm-credentials
- agent: kubernetes
  cluster: app.ci
  decorate: true
  extra_refs:
  - base_ref: master
    org: openshift
    repo: release
  interval: 12h
  labels:
    ci.openshift.io/role: infra
  name: periodic-openshift-release-master-newCluster-apply
  spec:
    containers:
    - args:
      - --config-dir=clusters/build-clusters/newCluster
      - --as=
      - --KUBECONFIG=/etc/build-farm-credentials/KUBECONFIG
      - --confirm=true
      command:
      - applyconfig
      image: applyconfig:latest
      imagePullPolicy: Always
      name: ""
      resources:
        requests:
          cpu: 10m
      volumeMounts:
      - mountPath: /etc/build-farm-credentials
        name: build-farm-credentials
        readOnly: true
    serviceAccountName: config-updater
    volumes:
    - name: build-farm-credentials
      secret:
        items:
        - key: sa.config-updater.newCluster.config
          path: kubeconfig
        secretName: build-farm-credentials
